pkg/cke/main.go

```diff
@@ -98,9 +99,25 @@ func main() {
 	if err != nil {
 		log.ErrorExit(err)
 	}
-	controller := server.NewController(session, interval, gcInterval, timeout, addon)
 
+	// Controller
+	controller := server.NewController(session, interval, gcInterval, timeout, addon)
 	well.Go(controller.Run)
+
+	// Metrics
+	collector := cke.NewCollector(etcd)
+	metricsHandler := cke.GetHandler(collector)
+	mux := http.NewServeMux()
+	mux.Handle("/metrics", metricsHandler)
+	ms := &well.HTTPServer{
+		Server: &http.Server{
+			Addr:    *flgMetrics,
+			Handler: mux,
+		},
+	}
+	ms.ListenAndServe()
+
+	// API server
```

Comment:
Why don't you merge the two?

Reply:
I supposed that the REST API of CKE was too complicated to be multiplexed with `/metrics`.  Actually CKE has just two endpoints.  Merged.